python/lsst/atmospec/utils.py

```diff
@@ -279,25 +276,6 @@ def rotateExposure(exp, nDegrees, kernelName='lanczos4', logger=None):
     return rotatedExp
 
 
-def airMassFromRawMetadata(md):
-    """Calculate the visit's airmass from the raw header information.
-
-    Returns the airmass, or 0 if the calculation fails.
-    Zero was chosen as it is an obviously unphysical value, but means
-    that calling code doesn't have to test if None, as numeric values can
-    be used more easily in place."""
-    time = Time(md['DATE-OBS'])
-    if md['RASTART'] is not None and md['DECSTART'] is not None:
-        skyLocation = SkyCoord(md['RASTART'], md['DECSTART'], unit=u.deg)
-    elif md['RA'] is not None and md['DEC'] is not None:
-        skyLocation = SkyCoord(md['RA'], md['DEC'], unit=u.deg)
-    else:
-        return 0
-    altAz = AltAz(obstime=time, location=AUXTEL_LOCATION)
-    observationAltAz = skyLocation.transform_to(altAz)
-    return observationAltAz.secz.value
-
-
```

Comment:
This function is used outside this package, so even though you've removed the need for it here, could you reinstate this please?

Reply:
Replaced with three lines of code using `ObservationInfo`.